fortuna/src/backend/routes/api/signup.js

```diff
@@ -0,0 +1,38 @@
+//@flow strict
```

Comment:
Did you make the changes to this file? If not, they shouldn't be in this PR. If someone else made this in some other commit, they should push those themself in a separate PR. If we conglomerate all the changes into one branch and push that, then we lose track of who wrote which code, so we can't keep track of who knows how what works, et cetera